[2025-05-04T19:54:03.486+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_load.wait_clients_csv manual__2025-05-04T19:53:58.338700+00:00 [queued]>
[2025-05-04T19:54:03.493+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_load.wait_clients_csv manual__2025-05-04T19:53:58.338700+00:00 [queued]>
[2025-05-04T19:54:03.493+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-05-04T19:54:03.504+0000] {taskinstance.py:1327} INFO - Executing <Task(FileSensor): wait_clients_csv> on 2025-05-04 19:53:58.338700+00:00
[2025-05-04T19:54:03.509+0000] {standard_task_runner.py:57} INFO - Started process 4782 to run task
[2025-05-04T19:54:03.511+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract_load', 'wait_clients_csv', 'manual__2025-05-04T19:53:58.338700+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/extract_load.py', '--cfg-path', '/tmp/tmp6cnd_m_c']
[2025-05-04T19:54:03.512+0000] {standard_task_runner.py:85} INFO - Job 132: Subtask wait_clients_csv
[2025-05-04T19:54:03.523+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-04T19:54:03.549+0000] {task_command.py:410} INFO - Running <TaskInstance: extract_load.wait_clients_csv manual__2025-05-04T19:53:58.338700+00:00 [running]> on host 773728ee9afd
[2025-05-04T19:54:03.625+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='finbest' AIRFLOW_CTX_DAG_ID='extract_load' AIRFLOW_CTX_TASK_ID='wait_clients_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T19:53:58.338700+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T19:53:58.338700+00:00'
[2025-05-04T19:54:03.641+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2025-05-04T19:54:03.642+0000] {filesystem.py:64} INFO - Poking for file /data/clients.csv
[2025-05-04T19:54:03.642+0000] {filesystem.py:69} INFO - Found File /data/clients.csv last modified: 20250501214538
[2025-05-04T19:54:03.642+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2025-05-04T19:54:03.651+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=extract_load, task_id=wait_clients_csv, execution_date=20250504T195358, start_date=20250504T195403, end_date=20250504T195403
[2025-05-04T19:54:03.682+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-05-04T19:54:03.719+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
